src/lms-harmonizer/edfi_lms_harmonizer/scripts/mssql/0014_view_missing_assignment_submissions.sql

```diff
@@ -0,0 +1,11 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE VIEW lmsx.missing_assignment_submissions AS
```

Comment:
Can we choose a different view name? At a quick glance, I would interpret this as a query for students who have not turned in an assignment.